backend/apps/organizations/migrations/0034_migrate_org_groups.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+
+from apps.permissions.constants import ORG_ADMIN_TYPE, ORG_MEMBER_TYPE
+
+
+def migrate_org_groups(apps, schema_editor):
+    """Rename hr_group to admin_group and primary_group to member_group"""
+
+    Organization = apps.get_model("organizations", "Organization")
+
+    for org in Organization.objects.all():
+        if org.hr_group is not None:
+            org.hr_group.group_type = ORG_ADMIN_TYPE
+        if org.primary_group is not None:
+            org.primary_group.group_type = ORG_MEMBER_TYPE
```

Comment:
I think you need to call `org.hr_group.save()` and `org.primary_group.save()` after each of these, to atually persist the changes to the database. Also, do `hr_group` and `primary_group` exist anymore when you call them here, or should you rather use `admin_group` and `member_group`?

Reply:
Also, should you also update the groups' `name` to `ORG_ADMIN_GROUP_NAME` and `ORG_MEMBER_GROUP_NAME`, respectively?